sprint8/tema3.js

```diff
@@ -0,0 +1,8 @@
+function reverseArray(numbers) {
```

Comment:
```suggestion
function getReversesArray(numbers) {
```

De asemenea. Optional, dar se citeste mai usor